crawler/adapters/twitter.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'twitter'
+
+module Adapters
+  class Twitter < Base
+    def fetch(hashtag:, amount:)
+      client.search("#{hashtag}", result_type: :recent).take(amount).map do |tweet|
+        Post.new tweet.text, build_url(tweet.url), :twitter
+      end
+    end
+
+    def build_url(url)
```

Comment:
I think we can move this to the `private` block too unless you're using it outside of here